packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -1666,57 +1708,51 @@ namespace Ifpack2 {
                    default_mode_type,KB::Algo::LU::Unblocked>
               ::invoke(member, A, tiny);
           }
-        }
+        } else {
+	  // for block jacobi invert a matrix here
+	  auto W = Kokkos::subview(WW, Kokkos::ALL(), Kokkos::ALL(), v);
+	  KB::Copy<member_type,KB::Trans::NoTranspose,default_mode_type>
```

Comment:
Future revisions should consider a "team BLAS" - like free function interface here, that calls `KB::Copy::invoke` etc.

Reply:
We have TeamCopy and SerialCopy. This Copy function has selective mode "Serial/Team". Here if the execution space is host space then I use serial function and if execution space space is cuda, it calls team interface accordingly. This is my way to pursue performance for both KNL and GPU architectures.